### DISCUSSION TITLE
A minimal product that deploys dioxus-web pages to Vercel

### DISCUSSION THREAD

**Discussion by lucifer1004** [DioxusLabs/dioxus#65] [Show and tell]
I made a minimal product ([dioxus-vercel-demo](https://github.com/lucifer1004/dioxus-vercel-demo)) that deploys dioxus-web pages to Vercel and would like to share it with the community in case anyone wants to do this.

**Comment by jkelleyrtp** [comment]
This is amazing! Can we add this to two repos:

- http://github.com/DioxusLabs/example-projects
- http://github.com/DioxusLabs/awesome-dioxus

Maybe with a little comment or two about what it does?

**Reply by lucifer1004** [reply]
Sure.

**Reply by lucifer1004** [reply]
I have polished the repo.
